CONTRIBUTING.md

```diff
@@ -30,4 +30,16 @@ This project uses ZenHub for tracking of issues and roadmap.
 
 ## Reporting an Issue
 
-To report an issue, or to suggest an idea for a change, open an [issue](../../issues/new). It is best to check our existing [issues](../../issues) first to see if a similar one has already been opened and discussed.
\ No newline at end of file
+To report an issue, or to suggest an idea for a change, open an [issue](../../issues/new). It is best to check our existing [issues](../../issues) first to see if a similar one has already been opened and discussed.
+## Getting Feedback on your contribution
+
+When creating a PR for one of our projects it is important to get your work reviewed by other members of the team. Below are some steps to getting your work reviewed. <br>
+* Assign a Reviewer from the list of reviewers on the project
```

Comment:
Maybe a first step could be to move the PR to the `Ready For Review` column on the sprint board so that the team can see that it is ready to be reviewed? 
Also you could add a bit more info to the other steps. Something like this
```suggestion
* Assign a Reviewer from the list of reviewers on the project
* Once the PR is reviewed the reviewer may request changes to be made, or if they are happy with the PR they will apply an `lgtm` label.
* When an `lgtm` label has been applied you can assign an Approver to your PR.
* The approver may request more changes, or if they are happy with the PR they will apply the `approve` label. After this time the PR will be merged.
```